_includes/main.liquid

```diff
@@ -118,15 +118,15 @@
         <section class="cta-subscribe">
             <div class="container">
                 <div class="subscribe-header">
-                    <h1 class="h1 cta-subscribe-h1">Stay updated on the latest Chainlink news</h1>
+                    <h1 class="h1 cta-subscribe-h1">Subscribe to the latest Chainlink Developer news</h1>
                 </div>
                 <div class="form-subscribe-wrapper w-form">
                     <form id="wf-form-Chainlink-Newsletter" name="wf-form-Chainlink-Newsletter" data-name="Chainlink Newsletter" class="form-subscribe">
                         <div class="form-subscribe-field-wrapper"><label for="Email" class="field-label-hidden">Email Address</label><input type="email" class="cta-subscribe-input w-input" maxlength="256" name="Email" data-name="Email" placeholder="Enter your email address" id="Email" required=""><input id="subscribe-button" type="submit" value="Sign up" class="btn btn-secondary w-button"></div>
                     </form>
                     <div class="form-success-message w-form-done">
                         <div class="subscribe-success-message-text">Thank you for signing up! Please check your inbox to confirm your subscription.</div>
-                            <div class="subscribe-success-social">
+                            <div class="subscribe-success-social
```

Comment:
Add back

Reply:
The whole HTML was invalid previously. Fixed.